### ISSUE TITLE
SpaceBetweenParens works in comments

### ISSUE THREAD

**Issue by za-creature** [brigade/scss-lint#475]
```scss
/*
( wat )
*/
```

```bash
> scss-lint --include-linter SpaceBetweenParens wat.scss
wat.scss:2 [W] SpaceBetweenParens: Expected 0 spaces between parentheses instead of 1
wat.scss:2 [W] SpaceBetweenParens: Expected 0 spaces between parentheses instead of 1
> scss-lint --version
scss-lint 0.39.0
```

There was no `.scss-lint.yml` file present in the working folder or any of its parents

**Comment by srawlins** [comment]
IMHO, SpaceBetweenParens looks like it should be re-implemented to only look at values, similar to SpaceAfterComma, or TrailingZero. The blanket regex implementation is a little scary.

**Comment by sds** [comment]
Fixed by #558.
